src/base/rss/rssfeed.cpp

```diff
@@ -399,6 +399,22 @@ void Feed::downloadArticleTorrentIfMatching(const ArticlePtr &article)
     BitTorrent::Session::instance()->addTorrent(torrentUrl, params);
 }
 
+QString Feed::generateSavePath(const QString &savePathTpl, const ArticlePtr &article)
+{
+    QRegExp rx("<(.+)>");
```

Comment:
yeah, I ran that code with the setMinimal(true) with such a case, and got the right output folder.